my-app/src/shared/components/VideoPlayer.js

```diff
@@ -0,0 +1,11 @@
+import { observer } from 'mobx-react';
+
+export const VideoPlayer = observer((videos) => {
+  return (
+    <iframe
+      width="420"
```

Comment:
please pass all of the properties from parent where you use this component, each and every shared component should not have any hardcoded values inside, since you'll be using them in different parts of the system